cmd/probe.go

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package cmd
+
+import (
+	"github.com/kubearmor/kubearmor-client/probe"
+	"github.com/spf13/cobra"
+)
+
+// versionCmd represents the get command
+var probeCmd = &cobra.Command{
```

Comment:
Let's update the comment to match the the command

Reply:
yes, It makes sense to have the enforcement check for the host to be in the same PR.